src/encord_active/server/main.py

```diff
@@ -11,7 +11,7 @@
 
 app.include_router(project.router)
 
-origins = ["http://localhost:5173", "http://localhost:8501", get_settings().ALLOWED_ORIGIN]
+origins = ["http://localhost:5173", "http://localhost:8501", "http://localhost:8000", get_settings().ALLOWED_ORIGIN]
```

Comment:
[curious]🤔

Not sure why we need this one

Reply:
i was having cors issues on my local test machine. I can remove from the pr.